packages/css/src/components/file-input/file-input.scss

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@import "../../common/text-rendering";
+
+@mixin reset-button {
+  border: 0;
+}
+
+.ams-file-input {
+  align-items: center;
+  color: var(--ams-file-input-color);
+  display: flex;
+  font-family: var(--ams-file-input-font-family);
+  font-size: var(--ams-file-input-font-size);
+  font-weight: var(--ams-file-input-font-weight);
+  justify-content: flex-start;
+  line-height: var(--ams-file-input-line-height);
+  outline-offset: var(--ams-file-input-outline-offset);
+  touch-action: manipulation;
+  width: 100%;
+
+  @include text-rendering;
+}
+
+.ams-file-input:disabled {
+  color: var(--ams-file-input-disabled-color);
+  cursor: not-allowed;
+}
+
+.ams-file-input::file-selector-button {
+  background-color: var(--ams-file-input-file-selector-button-background-color);
+  box-shadow: var(--ams-file-input-file-selector-button-box-shadow);
+  color: var(--ams-file-input-file-selector-button-color);
+  cursor: var(--ams-file-input-file-selector-button-cursor);
```

Comment:
Shouldn't `cursor: pointer` be defined for the whole input? The entire thing is clickable